src/OpenLoco/GameCommands/ChangeLandMaterial.cpp

```diff
@@ -0,0 +1,65 @@
+#include "../Map/MapGenerator.h"
+#include "../Map/TileLoop.hpp"
+#include "../Map/TileManager.h"
+#include "../Objects/LandObject.h"
+#include "../Objects/ObjectManager.h"
+#include "../S5/S5.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    // 0x00468EDD
+    static uint32_t changeLandMaterial(Map::Pos2 pointA, Map::Pos2 pointB, uint8_t landType, uint8_t flags)
+    {
+        GameCommands::setExpenditureType(ExpenditureType::Construction);
+        uint8_t x = (pointA.x + pointB.x) / 2 + 16;
```

Comment:
These should not be `uint8_t`. You can't use uint8_t to store coordinates in locomotion as they are not wide enough. I would prefer if you just did:
`const auto midPoint = (PointA + PointB) / 2 + Map::Pos2{ 16, 16 };`

Reply:
Good idea, thanks.